src/accessibility/context_menu/ContextMenuButton.tsx

```diff
@@ -36,7 +36,6 @@ export const ContextMenuButton = forwardRef(function <T extends keyof JSX.Intrin
             {...props}
             onClick={onClick}
             onContextMenu={onContextMenu ?? onClick ?? undefined}
-            title={label}
```

Comment:
why does this no  longer need a title?

Reply:
Previously, despite the title attribute, the browser wasn't displayed in this case. (For example the user menu (avatar) at the top left of EW). I removed the title attribute to keep the same behaviour with the new `AccessibleButton` change. In term of accessibility, the `aria-label` attribute is already provided.